contracts/Math.sol

```diff
@@ -152,7 +152,7 @@ library GnosisMath {
 
     /// @dev Returns maximum of an array
     /// @param nums Numbers to look through
-    /// @return Maximum number
+    /// @return maxNum Maximum number
```

Comment:
nit: revert?

Reply:
Not sure I get this comment.

Reply:
I think this is a NatSpec thing as well.

Reply:
I will also put a dash here to distinguish it.

Reply:
```suggestion
    /// @return maxNum - Maximum number
```

Reply:
Actually I wouldn't mind renaming this to just max as well...